kubernetes/cmsweb/hpa/dbs-hpa.yaml

```diff
@@ -59,7 +59,7 @@ spec:
     kind: Deployment
     name: dbs-phys03-r # should match deployment target
   minReplicas: 5
-  maxReplicas: 8
+  maxReplicas: 10
```

Comment:
@muhammadimranfarooqi , I think Yuyi said that only global-r needs 8-10 range, the phys03-r/w instances can use 5-8. Again, please double-check with her and confirm.

Reply:
global-r/w needs 8-10 range,
phys03-r/w  5-8  range.

Reply:
@yuyiguo @vkuznet 

I have adjusted replicas.